src/source/query_features.js

```diff
@@ -1,12 +1,13 @@
 // @flow
 
+import {OverscaledTileID} from './tile_id.js';
```

Comment:
This is only used as type — I thought we had some lint rule about this? If not, we probably should.

```suggestion
import type {OverscaledTileID} from './tile_id.js';
```